h/services/bulk_executor/_executor.py

```diff
@@ -30,7 +33,18 @@ def __init__(self, db, authority="lms.hypothes.is"):
     def configure(self, config):
         """Process a configuration instruction."""
         self._assert_authority("effective user", config.effective_user, embedded=True)
-        self.effective_user = config.effective_user
+
+        try:
+            user = (
+                self.db.query(User).filter(User.userid == config.effective_user).one()
+            )
+
+        except NoResultFound:
+            raise InvalidDeclarationError(
+                f"No user found for effective user: '{config.effective_user}'"
+            )
+
+        self.effective_user_id = user.id
```

Comment:
This is the first time we actually make use of the `effective_user` to set the group creator, so we actually need to look it up now.